package.json

```diff
@@ -12,6 +12,7 @@
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
+    "react-native-maps": "^0.27.1",
```

Comment:
Didn't realize there was a specific maps component for react native, this is perfect!

Reply:
Yeah super dope 😎